crontab/del_teams.php

```diff
@@ -5,21 +5,24 @@
 
 require_localhost_request();
 
-$old_date = time() - 7776000;
-    
-$result = mysqli_query(DPDatabase::get_connection(), "SELECT * FROM user_teams WHERE active_members <= 1");
+// Delete teams created more than 90 days that have no members
```

Comment:
This crontab file isn't in the `SETUP/dp.cron` so it's not even set to be used. It also isn't used on PROD or do I intend on adding it. I updated it so that it's at least consistent with the new implementation.

We do not have an interface to delete forum topics or posts and given the above I don't think it's valuable to add them just for this unused crontab.